layout_test/assets/css/sidebar.css

```diff
@@ -0,0 +1,393 @@
+.sidebar {
```

Comment:
All the CSS properties are name-spaced to the `.sidebar` class to avoid collisions.